src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -1504,6 +1506,35 @@ public function normalizeId($id)
         return isset($this->definitions[$id]) || isset($this->aliasDefinitions[$id]) || isset($this->removedIds[$id]) ? $id : parent::normalizeId($id);
     }
 
+    /**
+     * Gets removed binding ids.
+     *
+     * @return array
```

Comment:
please keep only `@internal` in the docblock (no title) and use a real return type

Reply:
Are you sure with the return type, it’s against 3.4 branch, or am I mixing things up?

Reply:
oh, 3.4, my bad!